server/models/video/video-channel-sync.ts

```diff
@@ -0,0 +1,163 @@
+import { isVideoChannelSyncStateValid } from "@server/helpers/custom-validators/video-channel-syncs"
+import { CONSTRAINTS_FIELDS, VIDEO_CHANNEL_SYNC_STATE } from "@server/initializers/constants"
+import { VideoChannelSync, VideoChannelSyncState } from "@shared/models"
+import { MChannelSyncFormattable } from "@server/types/models/video/video-channel-sync"
+import { AttributesOnly } from "@shared/typescript-utils"
+import {
+  AllowNull,
+  BelongsTo,
+  Column,
+  CreatedAt,
+  DataType,
+  Default,
+  ForeignKey,
+  Is,
+  Model,
+  Table,
+  UpdatedAt
+} from "sequelize-typescript"
+import { AccountModel } from "../account/account"
+import { getSort, throwIfNotValid } from "../utils"
+import { VideoChannelModel } from "./video-channel"
+import { ActorModel } from "../actor/actor"
+import { Op } from "sequelize"
+import { UserModel } from "../user/user"
+import { isVideoImportTargetUrlValid } from "@server/helpers/custom-validators/video-imports"
+
+type AvailableForListOptions = {
+  accountId: number
+  search?: string
+  host?: string
+  handles?: string[]
+  forCount?: boolean
+}
+
+@Table({
+  tableName: 'videoChannelSync'
```

Comment:
Add an index on `videoChannelId`

Reply:
Ahah! I wondered why I didn't see any index along with `create table` instructions. That's because they're declared in models!